src/float_cast.h

```diff
@@ -106,17 +106,40 @@
 
 		return intgr ;
 	}
-#elif (defined (WIN32) || defined (_WIN32)) && defined(_M_3X64)
+#elif (defined (WIN32) || defined (_WIN32)) && defined(_M_X64)
 
 	#include <math.h>
 	#include <immintrin.h>
+	#include <emmintrin.h>
 
-	__inline long int
-	lrintf (float flt)
+	#ifdef _MSC_VER
+		#pragma function(lrint, lrintf)
+	#endif
+
+	__inline 
+	long int lrint(double flt)
```

Comment:
Argument might be called "arg" consistently with en.cppreference.com documentation, to avoid confusion

Reply:
I was actually copying the signature from the x86 code